testutil/logging.go

```diff
@@ -0,0 +1,20 @@
+package testutil
+
+import (
+	"testing"
+
+	"github.com/go-kit/kit/log"
+)
+
+type logger struct {
+	t *testing.T
+}
+
+func NewLogger(t *testing.T) log.Logger {
```

Comment:
go-lint : exported func requires a comment.

Reply:
We don't seem to require go-lint in this repo, there are lots of existing warning in the codebase.  If we do, can I suggest we add it to CI?  Either was I'll add a comment here.